app/views/group_offers/index.xlsx.axlsx

```diff
@@ -0,0 +1,73 @@
+wb = xlsx_package.workbook
+
+col_header = wb.styles.add_style(:bg_color => "FFDFDEDF", b: true , alignment: { horizontal: :center, vertical: :center }, border: {
+  color: '00', edges: [:bottom], style: :thin
+})
+
+wb.add_worksheet(name: t('group_offers', count: 2)) do |sheet|
+  header_row = [
+    t_attr(:title),
+    t_attr(:location),
+    t_attr(:availability),
+    t_attr(:target_group),
+    t_attr(:duration),
+    t_attr(:offer_state),
+    t_attr(:volunteers),
+    t_attr(:group_offer_category)
+  ]
+
+  sheet.add_row header_row, style: col_header
+
+  cell_types = Array.new(header_row.size).map { :string }
+  cell_styles = Array.new(header_row.size).map { nil }
+  @group_offers.each do |offer|
+    location = "#{offer.organization} #{offer.location}" if offer.external?
+    location = "#{offer.department.contact.last_name}" if !offer.external? && !offer.department.nil?
```

Comment:
I would combine the conditions:

```ruby
location =
  if offer.external?
    "#{offer.organization} #{offer.location}"
  elsif offer.department
    "#{offer.department.contact.last_name}"
  else
    # what should happen in this case?
  end
```

Maybe it would also make sense to move this into a method on `GroupOffer`